fake/src/faker/mod.rs

```diff
@@ -226,3 +235,11 @@ pub mod automotive {
     }
 }
 
+pub mod uuid {
+    def_fakers! {
```

Comment:
seems no impl for V1, V2, V3, V4 in fake/src/faker/impls/uuid.rs?
def_faker macro will generate config struct, then you implement Dummy for these structs.